### PR TITLE
Final 0.80.0 tweaks

### PR DESCRIPTION
**Pull request by kflynn**
- Improve `Ingress` docs
- Make sure that Ambassador will do the right thing if we don't have permission to read `Ingress` resources (much like we do with CRDs)

### CODE REVIEW

File: docs/user-guide/ingress-controller.md [hunk 1]
```diff
@@ -1,86 +1,289 @@
...
+### When should I use an `Ingress` instead of annotations or CRDs?
+
+As of 0.80.0, Datawire recommends that Ambassador be configured with CRDs. The `Ingress`
+resource is available to users who need it for integration with other ecosystem tools, or
+who feel that it more closely matches their workflows -- however, it is important to 
+recognize that the `Ingress` resource is rather more limited than the Ambassador `Mapping`
+is (for example, the `Ingress` spec has no support for rewriting or for TLS origination).
+**When in doubt, use CRDs.**
+
+In the future, we expect to recommend an expanded role for the `Ingress` resource, with
```
Comment (richarddli) [inline docs/user-guide/ingress-controller.md]: Let's delete this paragraph

File: docs/user-guide/ingress-controller.md [hunk 2]
```diff
@@ -1,86 +1,289 @@
...
-
-   Yes, you can! Under the hood, Ambassador simply converts Ingress resources to Ambassador Mappings and TLS Contexts - so yes, different form of inputs can co-exist.
\ No newline at end of file
+Starting with version 0.80.0, Ambassador can act as a Kubernetes
+[ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/),
+reading configuration data from Kubernetes 
+[`Ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources.
+This makes it easier to work with other `Ingress`-oriented tools within the Kubernetes
+ecosystem (e.g. external DNS), and it makes it easier for users migrating from other
+ingress controllers to try Ambassador.
```
Comment (richarddli) [inline docs/user-guide/ingress-controller.md]: We should delete the reference to external DNS

**Comment by kflynn** [comment]
@richarddli Recheck? I updated the doc.

**Comment by kflynn** [comment]
Cheating since I know that @richarddli and @nbkrause looked at the docs, and @ark3 looked at the code bits...

**Pull request merged**